spec/app/tournament/tournament.spec.js

```diff
@@ -0,0 +1,121 @@
+import {Tournament} from "../../../app/tournament/tournament";
+import {Player} from "../../../app/tournament/player";
+
+describe('class Tournament', function() {
+
+    var spiderman = new Player("Spiderman", 1);
+    var batman = new Player("Batman", 3);
+    var theThing = new Player("The Thing", 5);
+
+    var tournament;
+
+    beforeEach( function() {
+        tournament = new Tournament();
+    });
+
+    function setMockedTournament(players) {
+        while (players--) {
+            tournament.addPlayer(spiderman);
+        }
+    }
+
+    it('should return false when searching for a non-enrolled player', function() {
+        expect( tournament.existsPlayer(spiderman) ).toBeUndefined();
```

Comment:
In JS, undefined is a valid false value, so you should not write a test where you expect `undefined`, you should expect *any false value*, not exactly `false`.